bootstrap.sh

```diff
@@ -67,7 +67,7 @@ echo
 echo "⏩ Performing system checks"
 
 is_installed docker
-is_installed docker-compose
+is_installed docker compose
```

Comment:
This will not work as intended, this just checks if `docker` is installed again.

I've opened another PR which handles this part:
* https://github.com/juliushaertl/nextcloud-docker-dev/pull/210

This is the only change:

```suggestion

(
  (docker compose ps --help >/dev/null 2>&1 && echo "✅ Docker Compose plugin is properly installed" | indent) ||
	  (is_installed docker-compose)
)

```

This will check if the Docker Compose plugin is installed, if not, then check if the old `docker-compose` command is available, else exit.

Reply:
I added your suggestion to my code, thank you.